gmplot/gmplot.py

```diff
@@ -125,6 +125,10 @@ def _process_kwargs(self, kwargs):
                             settings["edge_color"] or \
                             settings["face_color"]
 
+        settings["precision"] = kwargs.get("precision", None) or \
+                            kwargs.get("p", None) or \
+                            6
```

Comment:
Do we need "p" as an alias for precision? I think "c" was made an alias of "color" just to match matplotlib; can we say the same for precision? If not, then we can reduce this to:
```
settings["precision"] = kwargs.get("precision", 6)
```

Reply:
Accept.